benchmark/cuchembm/utils/smiles.py

```diff
@@ -30,7 +30,7 @@ def calculate_morgan_fingerprint(smiles, radius, nbits):
         mol = Chem.MolFromSmiles(smiles[i])
         if mol:
             fp = AllChem.GetMorganFingerprintAsBitVect(mol, radius, nBits=nbits)
-            fp = np.frombuffer(fp.ToBitString().encode(), 'b1')
+            fp = np.fromstring(fp.ToBitString(), 'u1') - ord('0')
```

Comment:
I wondered about this change last night but it was getting late to check if my memory was correct or not.